src/app/Library/CrudPanel/CrudFilter.php

```diff
@@ -21,6 +22,9 @@ class CrudFilter
 
     public function __construct($options, $values, $logic, $fallbackLogic)
     {
+        //make sure we use the camel cased version of name.
```

Comment:
Probably best to do this in 4.2 though. Just to be safe.

Reply:
Hey @tabacitu from what we discussed we should go back to the old way `do not touch filter names`, and to fix the problems with javascript function names and url parameters (fixing the stucked button etc) we would validate the filter name, so the name entered by developer must be valid otherwise we `abort()`.

We discussed doing this in 4.2 so we can document it properly.

Best,
Pedro